docs/language/learn-ql/javascript/type-tracking.rst

```diff
@@ -1,9 +1,11 @@
-Tutorial: API modelling using type tracking
-===========================================
+Using type tracking for API modeling
+====================================
 
-This tutorial demonstrates how to build a simple model of the Firebase API
+You can build a simple model of the Firebase API
```

Comment:
It looks as if Firebase is just used as an example, so suggest making this a little more general:

```suggestion
You can track data through an API by creating a model 
```